CHANGELOG.md

```diff
@@ -4,10 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1]
+### Added
+- CloudEvent exceptions and event type checking in http module ([#96])
+
 ## [1.0.0]
 ### Added
+- Update types and handle data_base64 structured ([#34])
 - Added a user friendly CloudEvent class with data validation ([#36])
 - CloudEvent structured cloudevent support ([#47])
+- Separated http methods into cloudevents.http module ([#60])
+- Implemented to_json and from_json in http module ([#72])
+
+### Fixed
+- Fixed top level extensions bug ([#71])
```

Comment:
I think we usually keep changelog PRs separate from other PRs.

Reply:
I'll revert this in that case